doc/fundamentals/append_overwrite.rst

```diff
@@ -1,571 +1,364 @@
-use this repo.
+   In the output above, you can see that Pachyderm created five split-files
+   from the original `count.txt` file. Each file has one line from the
+   original `count.txt`. You can check the contents of each file by
+   running the `pachctl get file` command. For example, to get
+   the contents of `count.txt/0000000000000000`, run the following
+   command:
 
-Now, take a one-line file containing
+   ```bash
+   $ pachctl get file raw_data@master:count.txt/0000000000000000
+   One
+   ```
+
+   This operation creates five datums that are processed by the
+   pipelines that use this repository as input.
+
+1. Create a one-line file called `more-count.txt` with the
+   following content:
 
-::
-   
+   ```bash
    Six
+   ```
 
-and load it into Pachyderm appending it to the count.txt file. If that
-file were named, “more-count.txt”, the command might look like
+1. Load this file into Pachyderm by appending it to the `count.txt` file:
 
-::
+   ```bash
+   $ pachctl put file raw_data@master:count.txt -f more-count.txt --split line
+   ```
 
-   pachctl put file line-data@master:my-data.txt -f more-count.txt --split line
+   **Note:**
+   If you do not specify `--split` flag while appending to
+   a file that was previously split, Pachyderm displays the following
+   error message:
 
-That will result in six files in the directory named “count.txt” in the
-input repo, each of which will have the following contents
+   ```bash
+   could not put file at "/count.txt"; a file of type directory is already there
+   ```
 
-::
+1. Verify that another file was added:
 
-   count.txt/0000000000000000: One
-   count.txt/0000000000000001: Two
-   count.txt/0000000000000002: Three
-   count.txt/0000000000000003: Four
-   count.txt/0000000000000004: Five
-   count.txt/0000000000000005: Six
+   ```bash
+   $ pachctl list file raw_data@master:count.txt
+   NAME                        TYPE SIZE
+   /count.txt/0000000000000000 file 4B
+   /count.txt/0000000000000001 file 4B
+   /count.txt/0000000000000002 file 6B
+   /count.txt/0000000000000003 file 5B
+   /count.txt/0000000000000004 file 5B
+   /count.txt/0000000000000005 file 4B
 
-This would result in one datum being processed in any pipelines that use
-this repo: the new file ``count.txt/0000000000000005``.
+   The `/count.txt/0000000000000005` file was added to the input
+   repository. Pachyderm considers
+   this new file as a separate datum. Therefore, pipelines process
+   only that datum instead of all the chunks of `count.txt`.
+
+1. Get the contents of the `/count.txt/0000000000000005` file:
+
+   ```
+   $ pachctl get file raw_data@master:count.txt/0000000000000005
+   Six
+   ```
 
-Overwriting files with –split
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+## Example: Overwriting Files with `–-split`
 
 The behavior of Pachyderm when a file loaded with ``--split`` is
 overwritten is simple to explain but subtle in its implications.
```

Comment:
```suggestion
overwritten is simple to explain but subtle in its implications. Most importantly, it can have major implications when new row rows are inserting within the file, as shown in the example below, as opposed to just being appended to the end. 
```